src/configurations/destinations/fb/db-config.json

```diff
@@ -98,6 +99,7 @@
         "ketchConsentPurposes"
       ],
       "reactnative": [
+        "useNativeSDK",
```

Comment:
@1abhishekpandey Do we need this ?
cc: @sandeepdsvs

Reply:
This field is defined for `android` and `iOS` (which are present in device mode). I also checked firebase (it's also supported as a device mode in react native) and over there also this field is added under `reactnative` (refer [here](https://github.com/rudderlabs/rudder-integrations-config/blob/05bd60831b0e31b049b5c1e7a6011fb9054c9b6e/src/configurations/destinations/firebase/db-config.json#L57)). Therefore, I added it.

Please let me know if you think it is not needed. Then, I'll revert it.

Reply:
Hey @sanpj2292,
Please let me know if I need to do anything!

Reply:
@sanpj2292 I don't think other SDK's started using `connectionMode`